packages/web/src/services/audio-player/AudioPlayer.ts

```diff
@@ -219,9 +221,11 @@ export class AudioPlayer {
       this.gainNode = null
       this.source = null
       this.audioCtx = null
-      this._initContext(/* shouldSkipAudioContext */ true)
-      this.audio.setAttribute('preload', 'none')
-      this.audio.setAttribute('src', forceStreamSrc)
+
+      this._initContext()
+      this.audio.preload = 'none'
+      this.audio.crossOrigin = 'anonymous'
```

Comment:
Does this fix the tracks that weren't playing?

Reply:
yea